tools/src/main/scala/org/scalanative/bindgen/Bindgen.scala

```diff
@@ -65,7 +66,8 @@ object Bindgen {
                                 packageName: Option[String] = None,
                                 excludePrefix: Option[String] = None,
                                 extraArg: Seq[String] = Seq.empty,
-                                extraArgBefore: Seq[String] = Seq.empty)
+                                extraArgBefore: Seq[String] = Seq.empty,
+                                config: Option[File] = None)
```

Comment:
This change looks unrelated since there is no setter.

Reply:
Oops, it's from different commit